source/user_guide/concepts.rst

```diff
@@ -0,0 +1,115 @@
+    
+    The concept of feature contribution is similar to the data science concept of "feature importance". However, 
+    Howso is robust against several common challenges (correlated features, multiple distinguishing features) faced by other feature importance tools, including the SHAP metric, which often lead to misleading results.
+
+
+- Determine confidence level of each decision.​ 
+
+    Because of its underlying IBL nature, Howso determines in which region of the original data a prediction is located. Depending on the density of data in that region, Howso can inform users of its confidence, or certainty,
+    in its prediction. Thus, if a prediction about a new data point is made based on a very dense region of original data, which contains a lot of information that influences the prediction, Howso's confidence
+    in the prediction will be high. In contrast, if a prediction is made about a point located in a very sparse region of the data, which does not contain much information from which Howso can make a decision, Howso's 
+    confidence will be low. In the heart disease patient example, if you are making a prediction about a patient who has very similar health information to many other patients with heart disease,
+    Howso will be confident in its decision of a positive prediction. In contrast, you might have a patient who is extremely young and fit, but with high cholesterol levels, whose information doesn't match the information
+    of many other patients. Even if Howso predicts a positive diagnosis for this young patient, because this patient's information is in such a sparse region of dataspace, Howso will tell you it has low confidence
+    in its prediction, indicating further investigation may be necessary.
+
+Conviction
+----------
+
+The connection between IBL and information theory becomes apparent in the concept of **conviction**, which is a measure of surprisal. Conviction is the computed ratio of actual information to expected information, with
+a range of zero to infinity. A conviction value of one is average, as the actual information is equal to the expected information. Convictions less than one indicate higher surprisal, as the actual information
+iss less than the expected information, while convictions greater than one indicate lower surprisal, since the actual information is higher than the expected information. 
+
+Actual information is the amount of information that a data point actually adds when it is evaluated. Expected information is what the value of a new data point should be on average, given
+all knowledge of the data around it. For example, say you are again trying to make heart disease predictions, given historical heart disease data and a new patient's data is being analyzed.
+This patient has features very similar to historical data with positive prediction status. Thus, its expected heart disease prediction value is positive. Its actual information, though, is the true information
```

Comment:
minor suggestion. referring to a patient as it sounds a bit weird. Alternatives - the case('s), the patient('s). Up to you